frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -1,58 +1,95 @@
 import React, { useState } from "react";
-import { Box, Button } from "@chakra-ui/react";
+import { Box } from "@chakra-ui/react";
 import CampDetails from "./CampDetails";
 import ScheduleSessions from "./ScheduleSessions";
 import RegistrationForm from "./RegistrationForm";
+import Stepper, { CampCreationPages } from "./StepperNavigation/index";
 
-enum CampCreationPages {
-  CampCreationDetailsPage,
-  ScheduleSessionsPage,
-  RegistrationFormPage,
-}
+const CampCreationPage = (): React.ReactElement => {
+  // All response state from the three page components.
+  const [campDetailsDummyOne, setCampDetailsDummyOne] = useState(false);
+  const [campDetailsDummyTwo, setCampDetailsDummyTwo] = useState(false);
+  const [campDetailsDummyThree, setCampDetailsDummyThree] = useState("");
+  const [scheduleSessionsDummyOne, setScheduleSessionsDummyOne] = useState(
+    false,
+  );
+  const [scheduleSessionsDummyTwo, setScheduleSessionsDummyTwo] = useState(
+    false,
+  );
+  const [registrationFormDummyOne, setRegistrationFormDummyOne] = useState(
+    false,
+  );
+  const [registrationFormDummyTwo, setRegistrationFormDummyTwo] = useState(
+    false,
+  );
+
+  // State to determine whether or not all required fields have been filled out.
+  // NOTE: This will depend on what type of state a page requires, i.e. determining
+  // if a checkbox is checked is different than determining if an input field is filled.
+
+  const isCampDetailsFilled =
```

Comment:
Each page is going to have a different definition of what's required to be filled (some may be optional fields that wouldn't show up here).

Reply:
Yeah i think we can even abstract this to be a function if needed. Although a constant works well for now